projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -927,7 +927,7 @@
 			.animated-label__label {
 				position: absolute;
 				z-index: 1;
-				top: var(--jetpack--contact-form--animated-top-offset);
```

Comment:
This var is now unused, so I guess it's ok to get rid of it.

Reply:
Done [31b59e5](https://github.com/Automattic/jetpack/pull/43635/commits/31b59e508b99dfd2783e7bdb6615f97ee69d8e3c)